README.md

```diff
@@ -3,3 +3,36 @@
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
 Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+## Built With
+
+- Semantic HTML & CSS
+- Github workflow
+- Linters(Lighthouse, stylelint, Webhint, eslint)
+
+## Author
+
+👤 **Abimbola Aminat Buari**
+
+- GitHub: [bimbolabuari](https://github.com/bimbolabuari)
+- Twitter: [bimbolabuari](https://twitter.com/bimbolabuari)
+- LinkedIn: [bimbolabuari](https://linkedin.com/in/bimbolabuari)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Code reviewer(s).
+
+## 📝 License
+
+[MIT licensed](./LICENSE).
```

Comment:
- [ ] Well done adding a professional README file to the repo. Let's just fix this link by adding a valid LICENSE to our repo as well. This will ensure that people know how they can use this project to build other projects of their own. Here's a [template provided by Microverse](https://github.com/microverseinc/readme-template/blob/master/MIT.md) that you can use. Just remember to customize it as well.